firstvoices/backend/resources/utils/import_export_widgets.py

```diff
@@ -81,3 +81,15 @@ def clean(self, value, row=None, **kwargs):
         if curr_value is None:
             raise ValidationError("Invalid value. Expected 'true' or 'false'.")
         return str(not curr_value)
+
+
+class TextListWidget(Widget):
+    """Import/export widget to return valid values for arrayFields from attributes
+    that can span multiple columns in the input csv."""
+
+    def __init__(self, prefix, *args, **kwargs):
+        self.prefix = prefix
+        super().__init__(*args, **kwargs)
+
+    def clean(self, value, row=None, **kwargs):
+        return [value for key, value in row.items() if key.startswith(self.prefix)]
```

Comment:
"startswith" is more open than what we specified, and I think this leaves us open to surprise bugs and confusion. E.g., this would import text from a column called "acknowledgments" instead of "acknowledgement". I think we should use a regex here instead to match the field names exactly the way they are specified in the knowledge base documentation about preparing the csv files for importing.